albumentations/augmentations/functional.py

```diff
@@ -450,6 +450,39 @@ def equalize(img, mask=None, mode="cv", by_channels=True):
     return result_img
 
 
+def move_tone_curve(img, low_shift, high_shift):
```

Comment:
Please wrap this function with `@preserve_shape` because `cv2.LUT` may remove last channel from gray image